src/modules/netinstall/NetInstallPage.cpp

```diff
@@ -0,0 +1,139 @@
+#include "NetInstallPage.h"
+
+#include "./widgets/groupselectionwidget.h"
+#include "ui_page_netinst.h"
+#include "GlobalStorage.h"
+#include "JobQueue.h"
+#include "utils/Logger.h"
+
+#include <QFile>
+#include <QMap>
+#include <QTextStream>
```

Comment:
I don't consider this an issue, just a minor nitpick for style: Qt includes should go after KF5 includes.